code/_onclick/hud/alert.dm

```diff
@@ -272,7 +275,7 @@ or shoot a gun to move around via Newton's 3rd Law of Motion."
 		return
 	if(!blood_target && !sac_complete)
 		cut_overlays()
-		finder.icon_state = "no_sacrifice"
+		finder.icon_state = blend(sacrifice, ICON_OVERLAY)
```

Comment:
This is not how you add an overlay, by the way. use `add_overlay` if you want to use an overlay.